CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to libbpf-rs
+
+### Anyone is welcome to make libbpf-rs better!
+
+Thank you for wanting to contribute to libbpf-rs. We are delighted to have you.
```

Comment:
Nit: Perhaps consider highlighting libbpf-rs throughout all sections to make it stand out somewhat more: `**libbpf-rs**`?